INSTALL.md

```diff
@@ -3,7 +3,7 @@
 
 ### Prerequisites
 
-#### Ubuntu
+#### Debian / Ubuntu
 
 ```bash
 sudo apt-get install git libzip-dev libsdl1.2-dev libsdl-image1.2-dev libcurl4-openssl-dev zlib1g-dev
```

Comment:
Can you append `imagemagick` to this list? I think Fedora needs `GraphicsMagick`, but I don't have an install readily available, so not sure.

Reply:
For sure! Working on it.

**About Fedora**: i don't use it too, it's legacy install instructions, but seems you are right, [read here](https://www.tecmint.com/install-imagemagick-in-linux/).

Reply:
Done! You was right about Fedora!